cartography/intel/aws/ec2.py

```diff
@@ -72,6 +84,32 @@ def get_ec2_vpcs(session, region):
     return client.describe_vpcs()
 
 
+def load_ec2_key_pairs(session, data, region, current_aws_account_id, aws_update_tag):
+    ingest_key_pair = """
+    MERGE (keypair:KeyPair:EC2KeyPair{keyname: {KeyName}, region: {Region}})
+    ON CREATE SET keypair.firstseen = timestamp()
+    SET keypair.keyfingerprint = {KeyFingerprint}, keypair.lastupdated = {aws_update_tag}
+    WITH keypair
+    MATCH (aa:AWSAccount{id: {AWS_ACCOUNT_ID}})
+    MERGE (aa)-[r:RESOURCE]->(keypair)
+    ON CREATE SET r.firstseen = timestamp()
+    SET r.lastupdated = {aws_update_tag}
+    """
+
+    for key_pair in data['KeyPairs']:
+        key_name = key_pair.get("KeyName", "")
+        key_fingerprint = key_pair.get("KeyFingerprint", "")
```

Comment:
I recommend changing this to

```
key_fingerprint = key_pair.get('KeyFingerprint')
```

This way the field will not show up on the Neo4j node if it is undefined.  I'd rather have the field not be there rather than as an empty string.  (I realize there's some inconsistencies in other parts of our code but I think going forward we should make missing fields `None` rather than the empty string.)
```